vscode-extension/package.json

```diff
@@ -120,19 +157,20 @@
   "main": "./out/extension",
   "scripts": {
     "vscode:prepublish": "npm run download-coursier && npm run compile",
-    "download-coursier":
-      "curl -L -o coursier https://github.com/coursier/coursier/raw/v1.0.0-RC13/coursier",
+    "download-coursier": "curl -L -o coursier https://github.com/coursier/coursier/raw/v1.0.0-RC13/coursier",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
     "postinstall": "node ./node_modules/vscode/bin/install",
     "test": "npm run compile && node ./node_modules/vscode/bin/test",
-    "build": "vsce package"
+    "prebuild": "cp ../README.md . && cp ../LICENSE LICENSE.md",
+    "build": "vsce package",
+    "postbuild": "rm README.md && rm LICENSE.md"
```

Comment:
these prebuild/postbuild tricks are needed to include README/LICENSE in the extension package, so that they're displayed on the Marketplace page.

I've tried other approaches involving relative paths and symlinks but none worked.